examples/Mass Groups & Collaborations Update/README.md

```diff
@@ -0,0 +1,132 @@
+# Mass Groups & Collaborations Update #
+
+## Overview ##
+The Mass Groups and Collaborations Update powershell script will use the Box CLI to create or update groups, add users to them and finally to create collaborations for groups to the folders.
+
+The entire script consists of two parts, each of which can be run separately or both.
+
+1. The first part is responsible for creating groups if they don't already exist, and adding users to them if they are not already there. It takes a .csv file from [GroupsUpdatePath](/examples/Mass%20Groups%20%26%20Collaborations%20Update/Mass_Groups_Collabs_Update.ps1#L19) as input. As you can see in an example csv file [Groups_Update.csv](/examples/Mass%20Groups%20%26%20Collaborations%20Update/Groups_Update.csv), each row defines a pair between a group and a user. So each group name can be used multiple times with different users, as well as user can be used multiple times with different groups.
```

Comment:
GroupsUpdate: suggest to rename that phase to Group users adding as you can only add new users to the group. And script won't delete any users which are already part of the group but not in the CSV.

Reply:
Change CSV name: Group_update -> User group addition. Keep description as is.